<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1006</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="divide">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>83</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="multiple">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>83</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="minus">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>230</y>
      <width>83</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>421</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>نمودار اول</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>110</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Y2=</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>X1=</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_1Y2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>110</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>50</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>X2=</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_1X2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>50</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_1Y1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>110</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_1X1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Y1=</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>421</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>نمودار دوم</string>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>60</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>X2=</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2X1">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2Y2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>120</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>120</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Y2=</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Y1=</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2X2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>60</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>X1=</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2Y1">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="sum">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>230</y>
      <width>83</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>40</y>
      <width>501</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1006</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
